<details>
<summary>Pytest Test failed! Click here</summary>

```
running test
Searching for setuptools>=40.3.0
Reading https://pypi.org/simple/setuptools/
Downloading https://files.pythonhosted.org/packages/cd/86/db096edd95e839f3fe123f78ae891efe9caf0e28f0333f2e9ac584bab9fd/setuptools-49.3.1-py3-none-any.whl#sha256=b90e630c5d6b118357392245a9a3b34d22e06fe94538b4b0830a7b4b693a0e5c
Best match: setuptools 49.3.1
Processing setuptools-49.3.1-py3-none-any.whl
Installing setuptools-49.3.1-py3-none-any.whl to /workspace/repo/.eggs
writing requirements to /workspace/repo/.eggs/setuptools-49.3.1-py3.6.egg/EGG-INFO/requires.txt

Installed /workspace/repo/.eggs/setuptools-49.3.1-py3.6.egg
running egg_info
creating thoth_report_processing.egg-info
writing thoth_report_processing.egg-info/PKG-INFO
writing dependency_links to thoth_report_processing.egg-info/dependency_links.txt
writing requirements to thoth_report_processing.egg-info/requires.txt
writing top-level names to thoth_report_processing.egg-info/top_level.txt
writing manifest file 'thoth_report_processing.egg-info/SOURCES.txt'
reading manifest file 'thoth_report_processing.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
warning: no files found matching '*.template'
warning: no files found matching '.coafile'
warning: no files found matching '*.html' under directory 'docs'
warning: no files found matching '*.py' under directory 'docs'
warning: no files found matching '*.rst' under directory 'docs'
warning: no files found matching '*.yaml' under directory 'tests'
writing manifest file 'thoth_report_processing.egg-info/SOURCES.txt'
running build_ext
inspection_test (unittest.loader._FailedTest) ... ERROR

======================================================================
ERROR: inspection_test (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: inspection_test
Traceback (most recent call last):
  File "/usr/lib64/python3.6/unittest/loader.py", line 153, in loadTestsFromName
    module = __import__(module_name)
  File "/workspace/repo/tests/inspection_test.py", line 22, in <module>
    from thoth.report_processing.components.inspection import AmunInspection, AmunInspectionsSummary
ImportError: cannot import name 'AmunInspection'


----------------------------------------------------------------------
Ran 1 test in 0.000s

FAILED (errors=1)
Test failed: <unittest.runner.TextTestResult run=1 errors=1 failures=0>
error: Test failed: <unittest.runner.TextTestResult run=1 errors=1 failures=0>
```
</details>

--- a/tests/inspection_test.py
+++ b/tests/inspection_test.py
@@ -41,7 +41,9 @@ class TestAdviser(ReportProcessingTestCase):
         )
 
         processed_inspection_runs = AmunInspections.process_inspection_runs(inspections_runs)
-        inspections_df = AmunInspections.create_inspections_dataframe(processed_inspection_runs=processed_inspection_runs)
+        inspections_df = AmunInspections.create_inspections_dataframe(
+            processed_inspection_runs=processed_inspection_runs
+        )
 
         dfs_inspection_classes, dfs_unique_inspection_classes = AmunInspectionsSummary.create_dfs_inspection_classes(
             inspection_df=inspections_df

--- a/tests/inspection_test.py
+++ b/tests/inspection_test.py
@@ -19,7 +19,7 @@
 
 from tests.base_test import ReportProcessingTestCase
 
-from thoth.report_processing.components.inspection import AmunInspection, AmunInspectionsSummary
+from thoth.report_processing.components.inspection import AmunInspections, AmunInspectionsSummary
 
 
 class TestAdviser(ReportProcessingTestCase):
@@ -29,19 +29,19 @@ class TestAdviser(ReportProcessingTestCase):
 
     def test_get_inspection_runs(self) -> None:
         """Test retrieving adviser results from local path."""
-        inspections_runs = AmunInspection.aggregate_thoth_inspections_results(
+        inspections_runs = AmunInspections.aggregate_thoth_inspections_results(
             repo_path=self._INSPECTIONS_FOLDER_PATH, is_local=True
         )
         assert inspections_runs
 
     def test_create_inspection_summary(self) -> None:
         """Test retrieving adviser results from local path."""
-        inspections_runs = AmunInspection.aggregate_thoth_inspections_results(
+        inspections_runs = AmunInspections.aggregate_thoth_inspections_results(
             repo_path=self._INSPECTIONS_FOLDER_PATH, is_local=True
         )
 
-        processed_data = AmunInspection.process_inspection_runs(inspections_runs)
-        inspections_df = AmunInspection.create_final_inspection_dataframe(processed_data=processed_data)
+        processed_inspection_runs = AmunInspections.process_inspection_runs(inspections_runs)
+        inspections_df = AmunInspections.create_inspections_dataframe(processed_inspection_runs=processed_inspection_runs)
 
         dfs_inspection_classes, dfs_unique_inspection_classes = AmunInspectionsSummary.create_dfs_inspection_classes(
             inspection_df=inspections_df

--- a/thoth/report_processing/components/inspection.py
+++ b/thoth/report_processing/components/inspection.py
@@ -551,10 +551,15 @@ class AmunInspections:
         # Runtime Environment:
         # Solver: OSName-OSVersion-PythonInterpreterVersion
         processed_string_result["solver"] = [pp[0] for pp in re_encoded]
+        processed_string_result["os_name"] = [solver[0] for solver in processed_string_result["solver"]]
+        processed_string_result["os_version"] = [solver[1] for solver in processed_string_result["solver"]]
+        processed_string_result["python_interpreter"] = [solver[2] for solver in processed_string_result["solver"]]
         processed_string_result["solver_string"] = [pp[1] for pp in re_encoded]
         processed_string_result["solver_hash_id"] = [pp[2] for pp in re_encoded]
         # Hardware:
-        # TODO: Add HW info in final results
+        processed_string_result["cpu_brand"] = [
+            cpu_brand[0] for cpu_brand in inspections_df[["hwinfo__cpu_info__brand_raw"]].values
+        ]
 
         # PI
         processed_string_result["pi_name"] = [pi_n[0] for pi_n in inspections_df[["stdout__name"]].values]
@@ -636,7 +641,10 @@ class AmunInspections:
         :param final_inspections_df: df for plots provided by `create_final_dataframe`.
         :param performance_packages: list of packages names
         """
-        return final_inspections_df[["identifier"] + performance_packages + ["solver"] + ["elapsed_time", "rate"]]
+        solver = ["os_name", "os_version", "python_interpreter"]
+        return final_inspections_df[
+            ["identifier"] + performance_packages + ["cpu_brand"] + solver + ["elapsed_time", "rate"]
+        ]
 
 
 class AmunInspectionsStatistics:
@@ -761,7 +769,7 @@ class AmunInspectionsSummary:
         "platform": ["hwinfo__platform"],
         "processor": ["cpu_type__is", "cpu_type__has"],
         "ncpus": ["hwinfo__cpu_type__ncpus"],
-        "family": ["runtime_environment__hardware__cpu_family"],
+        "family": ["runtime_environment__hardware__cpu_family", "hwinfo__cpu_info__brand_raw"],
         "requirements_locked": ["requirements_locked__default", "requirements_locked___meta"],
         "base_image": ["os_release__name", "os_release__version"],
         "script": ["script", "script_sha256", "@parameters", "stdout__name", "stdout__component"],

